<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addgoodwindow</class>
 <widget class="QMainWindow" name="addgoodwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>添加商品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancleBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>291</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>种类：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="typeEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>成本：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="costEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>售价：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="priceEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>库存：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="stockEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sureBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
